controllers/util/mcv_util.go

```diff
@@ -98,7 +100,9 @@ func (m ManagedClusterViewGetterImpl) GetNFFromManagedCluster(resourceName, reso
 	}
 
 	mcvViewscope := viewv1beta1.ViewScope{
-		Resource:  "NetworkFence",
+		Kind:      "NetworkFence",
+		Group:     csiaddonsv1alpha1.GroupVersion.Group,
+		Version:   csiaddonsv1alpha1.GroupVersion.Version,
```

Comment:
@BenamarMk suggests to keep Resource, as ACM is using this:
https://github.com/stolostron/multicloud-operators-subscription/blob/249ef1cd2a8999817fff3f8d441ad6b6cb17465e/pkg/controller/appsubsummary/appsubsummary_controller.go#L332

To me this looks like duplication of information that can end if mismatch or entering untested code paths. I'll discuss it with ACM folks.

Reply:
Quoting @xuezhaojun in #focum-mce slack channel:

> For MCV part, if you specify group,kind,version, then no need to keep resource, can you see the code here: [https://github.com/stolostron/multicloud-operators-foundation/blob/2d8d035cc2028e0[…]f30accc375f84cdf168ccb72/pkg/klusterlet/view/view_controller.go](https://github.com/stolostron/multicloud-operators-foundation/blob/2d8d035cc2028e0ef30accc375f84cdf168ccb72/pkg/klusterlet/view/view_controller.go#L102) The structure is:
>
> if scope.Resource == "" {
		gvk := schema.GroupVersionKind{Group: scope.Group, Kind: scope.Kind, Version: scope.Version}
		...
	} else {
		mapping, err := restutil.MappingFor(r.Mapper, scope.Resource)
		...
	}
>
>
> We only need (scope.Group + scope.Kind + scope.Verison) or (scope.Reasource), no need to keep both.